crypto/keys/src/ed25519.rs

```diff
@@ -21,18 +21,29 @@ use mc_crypto_digestible::{DigestTranscript, Digestible};
 use mc_util_from_random::FromRandom;
 use mc_util_repr_bytes::{
     derive_core_cmp_from_as_ref, derive_debug_and_display_hex_from_as_ref,
-    derive_into_vec_from_repr_bytes, derive_prost_message_from_repr_bytes,
     derive_repr_bytes_from_as_ref_and_try_from,
 };
 use rand_core::{CryptoRng, RngCore};
-use serde::{Deserialize, Serialize};
 use zeroize::Zeroize;
 
+#[cfg(feature = "alloc")]
+use alloc::{vec::Vec};
+
+#[cfg(feature = "alloc")]
+use mc_util_repr_bytes::derive_into_vec_from_repr_bytes;
+
+#[cfg(feature = "prost")]
+use mc_util_repr_bytes::derive_prost_message_from_repr_bytes;
+
+#[cfg(feature = "serde")]
+use serde::{self as serde, Deserialize, Serialize};
+
+
 // ASN.1 DER Signature Bytes -- this is a set of nested TLVs describing
 // a detached signature -- use https://lapo.it/asn1js/
 //
 // I'm not really sure if this is the correct way to do this, but I'm using
-// https://tools.ietf.org/html/rfc5912 as a reference. Unfortunately, digital
+// https://tools.ietf.org/html/rfc`912 as a reference. Unfortunately, digital
```

Comment:
this looks like a stray character
```suggestion
// https://tools.ietf.org/html/rfc912 as a reference. Unfortunately, digital
```